torch/_inductor/codecache.py

```diff
@@ -143,10 +143,15 @@ def is_gcc():
 
 
 def cpp_compile_command(input, output, include_pytorch=False):
-    if include_pytorch:
+    if include_pytorch or (config.cpp.simdlen and config.cpp.simdlen > 1):
         ipaths = cpp_extension.include_paths() + [sysconfig.get_path("include")]
         lpaths = cpp_extension.library_paths() + [sysconfig.get_config_var("LIBDIR")]
         libs = ["c10", "torch", "torch_cpu", "torch_python", "gomp"]
+        macros = ""
+        if config.cpp.simdlen == 16:
+            macros = " " + "-DCPU_CAPABILITY_AVX512"
+        elif config.cpp.simdlen == 8:
+            macros = " " + "-DCPU_CAPABILITY_AVX2"
```

Comment:
I guess you also need to check `config.cpp.simdlen is not None` again here since `include_pytorch is True` would also enter here without checking None of `config.cpp.simdlen`?